### PR TITLE
Register and login facebook

### PR DESCRIPTION
**Pull request by sirinthra-cc**
Save facebook id instead

### CODE REVIEW

File: webserver/apps/users/serializers.py [hunk 1]
```diff
@@ -78,3 +93,13 @@ def update(self, instance, validated_data):
...
         return instance
+
+
+def get_facebook_id(facebook_token):
+    url = 'https://graph.facebook.com/me?fields=id&access_token=' + facebook_token
+    response = requests.get(url)
+    if response.status_code == 200:
+        response_dict = ast.literal_eval(response.content.decode("utf-8"))
+        return True, response_dict['id']
+    print(response)
```
Comment (neungkl) [inline webserver/apps/users/serializers.py]: Consider remove this line

File: webserver/apps/users/views.py [hunk 2]
```diff
@@ -6,24 +6,24 @@
...
+    "credit_cards": [\n
+        \t{\n
+            \t"id": 10,\n
+            \t"card_number": "1",\n
+            \t"card_holder_name": "2",\n
+            \t"type": 1,\n
+            \t"expiry_date": "2017-05-16",\n
+            \t"verification_no": "1"\n
+        }\n
+    ]\n
```
Comment (neungkl) [inline webserver/apps/users/views.py]: Please remove `\t` characters

**Comment by neungkl** [comment]
After I see the error in CI. It seems like you forget to dump the data with newer version of migration. Please ask @JThanat or @futurizing for details.

**Comment by futurizing** [comment]
@sirinthra-cc you need to run
```sh
cd webserver
./dump_data.sh
```
to dump the migrated data into `dump.json`.

**Comment by JThanat** [comment]
Still need to fix card number

File: webserver/apps/users/views.py [hunk 3]
```diff
@@ -13,15 +13,15 @@
...
-            \t"verification_no": "1"\n
+        {\n
+            "id": 10,\n
+            "card_number": "1",\n
+            "card_holder_name": "2",\n
+            "type": 1,\n
+            "expiry_date": "2017-05-16",\n
+            "verification_no": "1"\n
         }\n
     ]\n
```
Comment (neungkl) [inline webserver/apps/users/views.py]: Not sure it is bug or something, why character `\n` is still be there?

Comment (neungkl) [reply webserver/apps/users/views.py]: Oh, it is just comment. My fault.

**Pull request merged**